docs/why.qmd

```diff
@@ -2,24 +2,313 @@
+
+```{python}
+t.group_by(["species", "island"]).agg(count=t.count()).order_by("count")
+```
+
+## Polars
+
+```{python}
+con = ibis.polars.connect()  # <1>
+
+t = con.read_parquet("penguins.parquet")
+t.limit(3)
+```
+
+1. Change only your connection to switch between backends.
+
+```{python}
+t.group_by(["species", "island"]).agg(count=t.count()).order_by("count")
+```
+
+## DataFusion
+
+```{python}
+con = ibis.datafusion.connect()  # <1>
+
+t = con.read_parquet("penguins.parquet")
+t.limit(3)
+```
+
+1. Change only your connection to switch between backends.
+
+```{python}
+t.group_by(["species", "island"]).agg(count=t.count()).order_by("count")
+```
+
+## PySpark
+
+```{python}
+con = ibis.connect("pyspark://")  # <1>
+
+t = con.read_parquet("penguins.parquet")
+t.limit(3)
+```
+
+1. Change only your connection to switch between backends.
+
+```{python}
+t.group_by(["species", "island"]).agg(count=t.count()).order_by("count")
+```
+
+:::
+
+## Who is Ibis for?
+
+Ibis for data engineers, data analysts, and data scientists to use directly with
+their data platform(s) of choice. It also has benefits for [data
+platforms](#ibis-for-data-platforms),
+[data organizations](#ibis-for-data-organizations), and
+[data library developers](#ibis-for-data-library-developers).
+
+### Ibis for data practitioners
+
+You can use Ibis at any stage of your data workflow, no matter your role.
+
+**Data engineers** can use Ibis to:
+
+- write and maintain complex ETL/ELT jobs
+- replace fragile SQL string pipelines with a robust Python API
+- replace PySpark with a more Pythonic API that supports Spark and many other
+ backends
+
+**Data analysts** can use Ibis to:
+
+- use Ibis interactive mode for rapid exploration
+- [create end-to-end analytics workflows](./posts/ibis-analytics/index.qmd)
+- work in a general-purpose, yet easy to learn, programming language without the
+ need for formatting SQL strings
+
+**Data scientists** can use Ibis to:
+
+- extract a sample of data for local iteration with a fast local backend
+- prototype with the same API that will be used in production
+- preprocess and feature engineer data before training a machine learning model
+
+### Ibis for data platforms
+
+Data platforms can use Ibis to quickly bring a fully-featured Python dataframe
+library with minimal effort to their platform. In addition to a great Python
+dataframe experience for their users, they also get integrations into the
+[broader Python and ML ecosystem](#ecosystem).
+
+Often, data platforms evolve to support Python in some sequence like:
 
-## Portability
+1. Develop a fast query engine with a SQL frontend
+2. Gain popularity and need to support Python for data science and ML use cases
+3. Develop a bespoke pandas or PySpark-like dataframe library and ML
+ integrations
 
-You can reuse the same code across different backends.
+This third step is where Ibis comes in. Instead of spending X money and Y time
```

Comment:
Were `X` and `Y` left in intentionally?  Maybe swap in for a generic measure?  "...instead of spending eyewatering amounts of money and time..."

Reply:
it was but rewording for clarity to "spending a lot of time and money"